### PR TITLE
Online doc for the copyright BP

### PR DESCRIPTION
**Pull request by oorestisime**
- Added routes and templates for the documentation in the copyright BP. (api, url and overview).
- Added links in the sources documentation to the copyright documentation

### CODE REVIEW

**Comment by oorestisime** [comment]
Well i took some more time and added the about and base template for the copyright BP (which are going to be used in case of a c.d.n deployment)

File: debsources/app/copyright/templates/copyright/doc_url.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+<h3>License search</h3>
+
+<h4>Checksum search</h4>
+<p>
+  The URL for a SHAsum search is:
+  <span class="url">{{ url_prefix
+  }}/sha256/?checksum=<strong>sha256</strong>&package=<strong>packagename</strong>&suite=<strong>jessie</strong></span>
+  Specifying a suite or apackage name is optional.
```
Comment (matthieucan) [inline debsources/app/copyright/templates/copyright/doc_url.html]: small fix: add a space in "apackage"

File: debsources/app/copyright/templates/copyright/doc_url.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  Both package name and version are required parameters.
+  <a href="{{ url_prefix }}file/gnubg/1.02.000-2/doc/gnubg/gnubg.html/">See example</a>
+</p>
+
+<h3>Package list</h3>
+
+<h4>Alphabetically</h4>
+
+<p>
+  <span class="url">{{ url_prefix }}/list/<strong>INT</strong></span>
```
Comment (matthieucan) [inline debsources/app/copyright/templates/copyright/doc_url.html:56]: what's int? worth to give details ;)

File: debsources/app/copyright/templates/copyright/doc_url.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+<p>
+  <span class="url">{{ url_prefix
+    }}/prefix/<strong>prefix</strong></span>
+  <a href="{{ url_prefix }}/prefix/libz">See example</a>
+</p>
+<p>
+  A suite parameter can be given to limit listing to a given suite.
+  <span class="url">{{ url_prefix
+    }}/prefix/<strong>prefix?suite=suite_name</strong></span>
```
Comment (matthieucan) [inline debsources/app/copyright/templates/copyright/doc_url.html]: "?suite=" shouldn't be < strong >'ed

**Comment by matthieucan** [comment]
Great work!
I commented inline a few suggestions.

Something else that confused me: the doc hierarchy. I'm not happy with the current situation: a link shared by two doc pages (the db structure one), part of the doc being the same (in overview), and general navigation between docs. What about a single page which lists subpages, with only one Overview, for instance? What do you think?
@zacchiro ?

**Comment by zacchiro** [comment]
On Tue, Jul 07, 2015 at 09:31:19AM -0700, Matthieu Caneill wrote:

> Something else that confused me: the doc hierarchy. I'm not happy with
> the current situation: a link shared by two doc pages (the db
> structure one), part of the doc being the same (in overview), and
> general navigation between docs. What about a single page which lists
> subpages, with only one Overview, for instance? What do you think?

The structure you suggest looks indeed more reasonable / better
organized to me as well.

## Thanks for the idea,

Stefano Zacchiroli  . . . . . . .  zack@upsilon.cc . . . . o . . . o . o
Maître de conférences . . . . . http://upsilon.cc/zack . . . o . . . o o
Former Debian Project Leader . . . . . @zacchiro . . . . o o o . . . o .
« the first rule of tautology club is the first rule of tautology club »

**Comment by oorestisime** [comment]
fixed suggestions and added a new commit for the single page doc and doc-overview.. (i ll squash both after review)..

**Comment by matthieucan** [comment]
Thanks! Everything is fine with me. Can you squash/rebase to master before I merge?

Oh and small detail: could you s/DebSources/Debsources/?

Also, @zacchiro @oorestisime : if we merge and deploy this, we'll have links to copyright-related pages reachable from s.d.n. I don't know if we want to annouce it more publicly first?

**Pull request closed without merging**